content/posts/gitops/index.md

```diff
@@ -0,0 +1,497 @@
+---
+title: GitOps
+author: Lena Fuhrimann / Mathis Kretz
+comments: true
+date: 2022-05-19
+---
+
+This blog post is the third part of a three-part series adapted from a GitOps webinar series which we co-produced together with our friends at [VSHN](https://www.vshn.ch/).
+
+In this third part, we will show you how you can combine Git which we introduced in part 1 with Infrastructure as Code, which we introduced in part 2. The combination of these two worlds will lead us to GitOps where we will show you concepts and technologies which enable you to operate your infrastructure and applications entirely using Git repositories. We will proceed in two steps, first showing you how to do GitOps with applications and then how to extend the same principles to your entire infrastructure stack.
+
+If you have questions, feel free to post them as comments on this blog post. If you would rather sit back and enjoy this part as a webinar, then you may head over to [the recording on YouTube](https://youtu.be/F4ZgpxBCL7s).
+
+# The Idea
+
+Before we dive into how we perform GitOps for applications and infrastructure, let’s first consider the general ideas and assumptions behind GitOps.
+
+![](./idea.svg)
```

Comment:
Please add descriptions to these images (in the `[]`).

Reply:
Agreed. Done.